lightllm/server/router/manager.py

```diff
@@ -204,6 +224,17 @@ def _send_to_detokenization_proc(self, batch: Batch, req_ans):
         self.send_to_detokenization.send_pyobj(batch_out)
         return
         
+    def _log_stats(self):
+        if not self.log_stat:
+            return
+
+        now = time.time()
+        if now - self.last_log_time > _LOGGING_INTERVAL_SEC:
+            self.last_log_time = now
+            print(f"Avg tokens(prompt+generate) per second: ", self.all_tokens/(now-self.start_time), " tokens/s, "
+                  f"Avg generate tokens per second: ", self.output_tokens/(now-self.start_time), " tokens/s ")
```

Comment:
`self.all_tokens` and `self.output_tokens` not reset here.

Reply:
> `self.all_tokens` and `self.output_tokens` not reset here.

Maybe no need to reset here in current  statistics way. This is not a periodic statistics way. I record the tokens across the whole serving life, and the self.start_time only be init once when `loop_for_fwd` be called. That we can calculate the average TPS on all prompt+generate tokens before. 

Of course we can do periodic statistics, for example calculate the average TPS in the time windows with 10s.

Reply:
ok I understand. This way, we'd better keep the server busy, or the statics would become less valuable.

Then would you please add some comment or something clarifying the difference between the two statistics ways?

> Of course we can do periodic statistics, for example calculate the average TPS in the time windows with 10s.

Yes. If we print both them, it will cause less confuse. Would you like to?

Reply:
> ok I understand. This way, we'd better keep the server busy, or the statics would become less valuable.
> 
> Then would you please add some comment or something clarifying the difference between the two statistics ways?
> 
> > Of course we can do periodic statistics, for example calculate the average TPS in the time windows with 10s.
> 
> Yes. If we print both them, it will cause less confuse. Would you like to?

done, chang to periodic statistics now :)